homeassistant/config_entries.py

```diff
@@ -911,6 +927,52 @@ async def async_step_unignore(self, user_input: Dict[str, Any]) -> Dict[str, Any
         """Rediscover a config entry by it's unique_id."""
         return self.async_abort(reason="not_implemented")
 
+    async def async_step_user(
+        self, user_input: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, Any]:
+        """Handle a flow initiated by the user."""
+        return self.async_abort(reason="not_implemented")
```

Comment:
We should make the frontend capture this key and translate it without requiring every integration to translate it

Reply:
It is mainly here for completeness and pleasing the CI for this method to exists.